### PR TITLE
travis: update jdk, ruby

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

**Comment by supsrinivas** [comment]
Go for it! 👍

File: .travis.yml [hunk 1]
```diff
@@ -11,16 +10,17 @@ env:
...
 
 jdk:
-  - oraclejdk8
+  - oraclejdk9
 
 services:
   - redis-server
 
 before_install:
+  - rm -rf "${TRAVIS_BUILD_DIR}/tmp/solr-test"
```
Comment (supsrinivas) [inline .travis.yml]: Where is the `TRAVIS_BUILD_DIR` defined?

Comment (dunn) [reply .travis.yml]: https://docs.travis-ci.com/user/environment-variables/#default-environment-variables

**Comment by dunn** [comment]
Well, apparently that log directory doesn't exist.

**Comment by dunn** [comment]
:))))))) https://github.com/cbeer/solr_wrapper/commit/6a3b732c72e8c946393b511be49f2d523d84fae9

**Comment by supsrinivas** [comment]
@dunn where are we on fixing this?

**Comment by dunn** [comment]
@supsrinivas  Still failing, and I can't figure out why

**Comment by supsrinivas** [comment]
Ticket Fix travis oraclejdk8ation causing build failure => [DIGREPO-946](https://help.library.ucsb.edu/browse/DIGREPO-946) got fixed in [this PR](https://github.com/ucsblibrary/alexandria/pull/184). Closing this PR.

**Pull request closed without merging**